### ISSUE TITLE
Not working, 5/23/21

### ISSUE THREAD

**Issue by BG1959** [austencm/youtube-auto-like#81]
__________________________
### Log
Sun May 23 2021
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36
YouTube variant: material
navigated: https://www.youtube.com/watch?v=qYOvV3Ij1oI
loading options...
...options loaded (disabled: false,like_what: all,like_when: percent)
liker initialized
not a watch page
stopped

**Comment by markygnlg** [comment]
[Thought I'd post this comment here too as I didn't know this new thread existed]:

2 points I need to mention:

there is now a v2.7.1 which was released about 24hrs ago as of this writing:
https://github.com/austencm/youtube-auto-like/releases

secondly, I too found it 'not working' (with the 50% setting) but if I pressed 'F5'/refresh or opened the video in another tab (same as refreshing the page) the like button would be lit up: ie. it works but the graphics of the thumb only displays if you press refresh (including using v2.7.1 on Opera)

Update: I just tested it & it does not work. ie. v2.7.1 was behaving as described above for about 2 hours but now it is not (note: this was posted by me '3 hours' after the previous post [in the original thread, I copy/pasted this into this current thread])
